rgw/v2/tests/s3_swift/reusables/rgw_orphan_list.py

```diff
@@ -0,0 +1,148 @@
+import json
+import logging
+import sys
+
+import v2.utils.utils as utils
+
+log = logging.getLogger()
+
+
+def check_gc():
+    """Checks RGW garbage collection and exits if orphans are found."""
+    log.info("Running radosgw-admin gc process...")
+    utils.exec_shell_cmd("radosgw-admin gc process")
```

Comment:
@yuva-gayam , could you include --include-all flag along with gc process. As, this provides more coverage.